sources/ingest/importers/location.py

```diff
@@ -197,6 +276,77 @@ def set_accessibility_shortages(
         for viewpoint in self.viewpoints:
             viewpoint.shortages = viewpoint_id_to_shortages.get(viewpoint.id, [])[:]
 
+    def fix_unknown_and_zero_shortcomings(self) -> bool:
```

Comment:
I would split this (rather long) function to multiple private functions just to make it more clean. 🍝

Reply:
Taking into account in another PR with refactoring